common/RenderTiles.hpp

```diff
@@ -191,14 +191,7 @@ namespace RenderTiles
                             deltaGen.compressOrDelta(pixmap.data(), offsetX, offsetY,
                                                      pixelWidth, pixelHeight,
                                                      pixmapWidth, pixmapHeight,
-                                                     TileLocation(
-                                                         tileRect.getLeft(),
-                                                         tileRect.getTop(),
-                                                         tileRect.getWidth(),
-                                                         tileCombined.getPart(),
-                                                         canonicalViewId,
-                                                         tileCombined.getEditMode()
-                                                         ),
+                                                     tiles[tileIndex],
```

Comment:
This looks very dangerous to me. TileDesc may easily have a wire-id set here in the desc._id. Most likely it will be different to the last usage.
In this case - we not only use more memory to store this - but we also completely disable the Delta cache, Deltas - burn client bandwidth and worse.

I would like to see a unit test with wire-ids set to different numbers to ensure this does not happen if we do this.

I would want to see a custom comparison function here.

Or - better, we could split the TileLocation concept out of TileDesc - and whack it in a base-class of that - and just continue to use TileLocation :-)